### ISSUE TITLE
Larger Canvas File size needed 100mb is just to small and performance isn't great. . . .

### ISSUE THREAD

**Issue by chronoite** [MosaicManufacturing/canvas-issues#272]
Individuals are buying your printers, and pallets for models printed with multicolor in all shapes and sizes. Your printers even have a build volume of 355mmx355mmx355mm, but yet the software that's required cant manage it. Even smaller items I have duplicated in an attempt to print 12 on one print bed crash canvas when I click the custom support button. Overall if I can't print large files or fill a print bed with smaller one it kills efficiency.

**Comment by moberli** [comment]
Yes the max 100MB is really a issue - as well the performance is not that good.
